lib/module/plugin.js

```diff
@@ -3,7 +3,7 @@ import Auth from './auth'
 import './middleware'
 
 // Active chemes
-<%= options.uniqueSchemes.map(path =>`import ${'scheme_' + hash(path)} from '${path}'`).join('\n') %>
+<%= options.uniqueSchemes.map(path =>`import ${'scheme_' + hash(path)} from '${process.platform==="win32"?path.replace(/\\/g,'\\\\'):path}'`).join('\n') %>
```

Comment:
Why not?
```
<%= options.uniqueSchemes.map(path =>`import ${'scheme_' + hash(path)} from '${path.replace(/\\/g, '/')}'`).join('\n') %>
```

Reply:
for save platform specific path, but yes, it's also true